src/enums/location.enum.ts

```diff
@@ -0,0 +1,3 @@
+export enum LocationEnum {
+  PLATEAUX = `Plateaux`,
```

Comment:
is this an english word?

Reply:
Yes:
https://www.wordreference.com/enfr/plateaux

Reply:
By the definition of it I do not understand though how it is appropriate...
But be the judge of that of course.

Reply:
It's just a location name. Like, it can even be temporary lol

It could be a made up word that it wouldn't change anything? ;)

This is a lore location name!